delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/routes/FilesRoutesSpec.scala

```diff
@@ -107,21 +105,100 @@ class FilesRoutesSpec
+      override def create(id: IdSegment, storageId: Option[IdSegment], projectRef: ProjectRef, entity: HttpEntity)(
+          implicit caller: Caller
+      ): IO[FileRejection, FileResource]                                                         = ???
+      override def createLink(
+          storageId: Option[IdSegment],
+          projectRef: ProjectRef,
+          filename: Option[String],
+          mediaType: Option[ContentType],
+          path: Uri.Path
+      )(implicit caller: Caller): IO[FileRejection, FileResource]                                = ???
+      override def createLink(
+          id: IdSegment,
+          storageId: Option[IdSegment],
+          projectRef: ProjectRef,
+          filename: Option[String],
+          mediaType: Option[ContentType],
+          path: Uri.Path
+      )(implicit caller: Caller): IO[FileRejection, FileResource]                                = ???
+      override def update(
+          id: IdSegment,
+          storageId: Option[IdSegment],
+          projectRef: ProjectRef,
+          rev: Int,
+          entity: HttpEntity
+      )(implicit caller: Caller): IO[FileRejection, FileResource]                                = ???
+      override def updateLink(
+          id: IdSegment,
+          storageId: Option[IdSegment],
+          projectRef: ProjectRef,
+          filename: Option[String],
+          mediaType: Option[ContentType],
+          path: Uri.Path,
+          rev: Int
+      )(implicit caller: Caller): IO[FileRejection, FileResource]                                = ???
+      override def tag(id: IdSegment, projectRef: ProjectRef, tag: Tag.UserTag, tagRev: Int, rev: Int)(implicit
+          subject: Subject
+      ): IO[FileRejection, FileResource]                                                         = ???
+      override def deleteTag(id: IdSegment, projectRef: ProjectRef, tag: Tag.UserTag, rev: Int)(implicit
+          subject: Subject
+      ): IO[FileRejection, FileResource]                                                         = ???
+      override def deprecate(id: IdSegment, projectRef: ProjectRef, rev: Int)(implicit
+          subject: Subject
+      ): IO[FileRejection, FileResource]                                                         = ???
+      override def fetchContent(id: IdSegmentRef, project: ProjectRef)(implicit
+          caller: Caller
+      ): IO[FileRejection, FileResponse]                                                         = IO.pure(
+        FileResponse(
+          "file.name",
+          ContentTypes.`text/plain(UTF-8)`,
+          1024,
+          IO.raiseError(error)
+        )
+      )
+      override def fetch(id: IdSegmentRef, project: ProjectRef): IO[FileRejection, FileResource] = ???
+      override private[files] def attributesUpdateStream(offset: Offset)                         = ???
+      override private[files] def updateAttributes(iri: Iri, project: ProjectRef)                = ???
+    }
```

Comment:
This is what I wanted to turn scalafmt off for. An alternative might be to put it in another file, it's pretty horrible boilerplate

Reply:
Strange. I checked out this branch to try it out and it works fine if I just add `// format: off` before line 128. It doesn't reformat any changes whether I use cmd+opt+L in IntelliJ or if I use `scalafmtAll` directly in sbt.

Reply:
Probable error 40

Reply:
My only explanation is that I was doing it inside of the files block, and perhaps that's not legitimate